rpcs3/Emu/Cell/Modules/cellRtc.cpp

```diff
@@ -1189,6 +1187,8 @@ error_code cellRtcGetCurrentSecureTick(vm::ptr<CellRtcTick> tick)
 
 	// TODO
 
+	tick->tick = 0xe01d003a63a000;
```

Comment:
You can use cellRtcGetCurrentTick() for now if you want.
The syscall here returns the hardware clock corrected for drift. It is unaffected by the date and time settings in the XMB/VSH settings, which makes it "secure" I guess.

Reply:
I'm not gonna add stuff I haven't personally confirmed and I'm not gonna look into this any more for now.